Makefile

```diff
@@ -262,13 +264,35 @@ deploy-operator: ## Deploys istio-workspace operator resources to defined OPERAT
 
 .PHONY: undeploy-operator
 undeploy-operator: ## Undeploys istio-workspace operator resources from defined OPERATOR_NAMESPACE
-	$(call header,"Undeploying operator to $(OPERATOR_NAMESPACE)")
+	$(call header,"Undeploying operator from $(OPERATOR_NAMESPACE)")
 	$(call process_template,deploy/istio-workspace/operator.yaml) | oc delete -n $(OPERATOR_NAMESPACE) -f -
 	$(call process_template,deploy/istio-workspace/role_binding.yaml) | oc delete -n $(OPERATOR_NAMESPACE) -f -
 	oc delete -n $(OPERATOR_NAMESPACE) -f deploy/istio-workspace/role.yaml
 	oc delete -n $(OPERATOR_NAMESPACE) -f deploy/istio-workspace/service_account.yaml
 	oc delete -n $(OPERATOR_NAMESPACE) -f deploy/istio-workspace/crds/istio_v1alpha1_session_crd.yaml
 
+.PHONY: deploy-operator-local
+deploy-operator-local: export OPERATOR_WATCH_NAMESPACE=$(OPERATOR_NAMESPACE)
+deploy-operator-local: ## Deploys istio-workspace operator resources to a single Namespace defined by OPERATOR_NAMESPACE
+	$(call header,"Deploying local operator to $(OPERATOR_NAMESPACE)")
+	export OPERATOR_WATCH_NAMESPACE=$(OPERATOR_NAMESPACE)
```

Comment:
Any reason why we have this here and in `275`?

Reply:
Oh, there are reasons... but none that are valid. https://github.com/Maistra/istio-workspace/pull/215/commits/649f78fbd08f09a429d47264d544de66ee8d4bce